features/mbedtls/mbed/wrapper.c

```diff
@@ -0,0 +1,29 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "hal/rng_api.h"
+
+#if defined(MBEDTLS_ENTROPY_HARDWARE_ALT) && defined(DEVICE_RNG)
+
+int mbedtls_hardware_poll( void *data, unsigned char *output, size_t len, size_t *olen ) {
```

Comment:
If we tie the RNG into the entropy source API, it becomes our only possible entropy source. Is that what we want to do?

I think the RNG driver should be the default entropy source, but do we want it to be the *only* one?